spec/support/factory_girl.rb

```diff
@@ -6,6 +6,8 @@
     password 'password'
     password_confirmation 'password'
     sequence(:username) { |n| "user#{n}" }
+    profile_photo { Rack::Test::UploadedFile.new(File.join(
+      Rails.root, '/spec/support/images/example.jpg')) }
```

Comment:
Expression at 10, 56 should be on its own line.<br>Indent the first parameter one step more than `File.join(`.